pkgs/arch/.SRCINFO

```diff
@@ -0,0 +1,18 @@
+pkgbase = clapper-git
+	pkgdesc = A GNOME media player built using GJS and powered by GStreamer with OpenGL rendering. Can also be used as a pre-made widget for Gtk apps.
+	pkgver = 0.0.0
+	pkgrel = 1
+	url = https://github.com/Rafostar/clapper
+	arch = any
+	license = GPL-3.0
+	makedepends = meson
+	makedepends = gjs
```

Comment:
We also use `python3` in meson build.

Reply:
meson depends on python, as you can see it builds on the buildservice

Reply:
OK